src/pkg.js

```diff
@@ -57,20 +55,23 @@ const updateVersion = ({ path, packageName, logger }) => (
             peerDependencies = {}
         }, version) => {
             if (dependencies.hasOwnProperty(packageName)) {
+                assert(!satisfies(version, dependencies[packageName]), `${packageName} latest version ${version} matches existing dependency ${dependencies[packageName]}`);
```

Comment:
This is why we use allSettled instead of all...its so that each repo being upgraded can safely throw uncaught exceptions without affecting the other upgrades.

https://github.com/porchdotcom/up-to-code/blob/2e17b3dd8e5444ae18bfa5958812d1cd8da9a4cb/src/index.js#L224